pulsar_data_collection/database_connectors/database_actions.py

```diff
@@ -0,0 +1,34 @@
+from abc import ABC, abstractmethod
+
+"""
+Factory that represents actions related to a database,
+i.e. make a connection, and send data
+"""
+
+
+class DatabaseActions(ABC):
+    """
+    Basic representation of database connections instance
+    """
+
+    @abstractmethod
+    def make_connection(self):
+        """Sets up the connection object"""
+
+    @abstractmethod
+    def write_data(self):
+        """Writes Data to the database"""
+
+    @abstractmethod
+    def param_dict(self):
```

Comment:
Minor comments: Follow the same function naming convention (apply where applicable)
Suggested:
create_params_dict()